.github/workflows/docker-tagged.yml

```diff
@@ -0,0 +1,47 @@
+name: Docker Tagged Build
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_NAMESPACE: "openfaas"
+      DOCKER_IMAGE_NAME: "nats-connector"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Get TAG
+        id: get_tag
+        run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          registry: docker.io
+
+      - name: Build image and push to container registies
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: ./
+          file: ./Dockerfile
+          push: true
+          tags: ${{ env.DOCKER_NAMESPACE }}/${{ env.DOCKER_IMAGE_NAME }}:${{ steps.get_tag.outputs.TAG }}
```

Comment:
Where are you setting these env-vars?

Reply:
There are defined earlier in config, it's effectively a mirror of the approach from Travis. See line 12

Reply:
I learned something, thank you.

In this instance I'd say just hard-code it please to: `ghcr.io/openfaas/nats-connector`